### PR TITLE
Portfolio: RWD - Desktop Version using Media Queries

### PR DESCRIPTION
**Pull request by peterdtitan**
Updated the portfolio build to using responsive web design processed. The following files were changed: 

- index.html (Used semantic tags)
- styles.css (added media queries)
- assets (added more assets to aid visual design)

Screenshot 2022-06-04 at 4 34 31 AM

Screenshot 2022-06-04 at 4 35 01 AM
4161-acf4-045c3213c44c.png">

Screenshot 2022-06-04 at 4 35 15 AM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -71,6 +73,7 @@ <h2 class="container job-title">I am a Software Developer</h2>
     <section class="works-header">
         <div class="works-title">
             <h1>My Recent Works</h1>
+            <hr>
```
Comment (Kandy-Peter) [inline index.html:76]: - [ ] Please, ensure that The Title `"My Recent Works"`  is aligned to the left and after there's a line that takes the rest of the page's width.
**1.You design**
image
2. Figma **design**
image

**Review by Kandy-Peter — CHANGES_REQUESTED** [review]
Hi @ManarOmari 

Good job so far! I know you have worked hard on making this milestone. There is an issue that you still need to work on to go to the next project but you are almost there!

almost-there-almost


## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] I would advise you to implement your cards the way that the featured project is followed by 2 rows with 3 projects each (6 projects in total).
- [x] It would be nice if after the project description there were some buttons like tags listing all the technologies that were used to build the project as designed in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
- [x] Please, ensure that when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears to the bottom of the image and takes the whole image width mentioned button should take you to the Live Demo of the project or to the GitHub repository.
**1. Check your design**
image
**2. Figma design**
image

- [x] REQUESTED: 
     -I would suggest you add more black colours to your social icons to stick the Figma design.
        image

**1. Figma design**

image

 
- [x] Kindly check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by peterdtitan** [comment]
Made design changes to match the given template: 
-  Updated the responsiveness of headers and other elements
- Made changes to image resizing and use grid layout to add more cards. 

Screenshot 2022-06-04 at 4 45 12 PM

Screenshot 2022-06-04 at 4 45 25 PM

**Review by algerina — APPROVED** [review]
Hi @peterdtitan , 
Good job so far! :fire: 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed, however use a descriptive feature branch:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 


 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by ManarOmari** [comment]
Hello , the project is :   Portfolio: mobile version
<https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md>
not for desktop and the code you have sent in images to update it not for
me

**Pull request merged**